### DISCUSSION TITLE
how to cache precache videos in reusable video list?

### DISCUSSION THREAD

**Discussion by ShyamKrishna122** [jhomlala/betterplayer#529] [Q&A]
I have a list of videos and I have using a reusable video list to display those videos and all the videos are played from a URL so it's taking time to load the video. So, can we precache the video so that it loads a little faster. If so how can we do that in a reusable video list?

**Comment by jhomlala** [comment]
You should pre cache with free controller. For example there're 3 controllers available in list and 2 of them are used to render video. You can use last free controller to pre cache video which will be shown later in the list. This feature requires modification of reusable video list but I believe it's doable.
